sellotape/sellotape_dj/main_app/admin.py

```diff
@@ -1,3 +1,27 @@
 from django.contrib import admin
+from django.contrib.auth.models import User
 
 # Register your models here.
+
+# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+class User(models.Model):
```

Comment:
You can't both import the user class with a `from X import Y` and then also call your own class User.

Chances are the OneToOne links you're setting up below might try to link your model to itself instead of to the Django users model.